book/duckietown-robotics-development/01_connecting_and_operating_a_Duckiebot/06_basic_duckiebot_operation.md

```diff
@@ -33,7 +33,6 @@ This interface is really useful for quick debugging and for moving around manual
 
 But this interface has its limits, as it hides everything that is actually running on the duckiebot. To better understand the duckiebot, let’s take a look at what is under the hood : we will use portainer.
 
```

Comment:
```suggestion
<end/>
```

Reply:
if you dont add that the exercise will include the following line, which we dont want